### PR TITLE
feat: otomi install chart skeleton 

### PR DESCRIPTION
**Pull request by mojtabaimani**
This is not ready just for getting your comments.

### CODE REVIEW

File: otomi-install/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: otomi-install
```
Comment (svatwork) [inline otomi-install/Chart.yaml:2]: I don't know about the others but I would just name it otomi. I think it's standard if I look at the naming of other open source charts (e.g. https://github.com/prometheus-community/helm-charts/tree/main/charts: alertmanager is just called alertmanager, prometheus just prometheus).

Comment (Morriz) [reply otomi-install/Chart.yaml:2]: yes please

Comment (Morriz) [reply otomi-install/Chart.yaml:2]: and can this folder be renamed "chart" so it is clear it is the chart that comes with this software/

File: otomi-install/templates/serviceaccount.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: '{{ .Release.Name }}'
+---
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (Morriz) [inline otomi-install/templates/serviceaccount.yaml:6]: please put this in `rbac.yaml`

**Review by j-zimnowoda — COMMENTED** [review]
LGTM
An idea for improvement:
We could enrich this chart by moving values.schema.json inside the chart.
> values.schema.json  # OPTIONAL: A JSON Schema for imposing a structure on the values.yaml file
Read more: https://helm.sh/docs/topics/charts/

**Comment by mojtabaimani** [comment]
> LGTM
> An idea for improvement:
> We could enrich this chart by moving values.schema.json inside the chart.
> 
> > values.schema.json  # OPTIONAL: A JSON Schema for imposing a structure on the values.yaml file
> > Read more: https://helm.sh/docs/topics/charts/

the otomi-core value-schema.json is outside of the chart folder, so the helm command do not have access to it. Should we make a copy of the value schema in the chart folder?

**Comment by Morriz** [comment]
> An idea for improvement:
> We could enrich this chart by moving values.schema.json inside the chart.

we can generate that in there yes.

**Review by j-zimnowoda — APPROVED** [review]

**Pull request merged**